packages/babel-helpers/src/helpers.ts

```diff
@@ -34,38 +34,45 @@ helpers.wrapAsyncGenerator = helper("7.0.0-beta.0")`
   }
 `;
 
-helpers.asyncToGenerator = helper("7.0.0-beta.0")`
```

Comment:
I'm not sure if the version should be modified.

I will add `new-version-checklis` after https://github.com/babel/babel/pull/15959 is merged.